src/EnrollWindowsHelloController.js

```diff
@@ -110,25 +126,75 @@ function (Okta, FormController, FormType, webauthn, Footer, WindowsHelloErrorMes
 
       formChildren: function () {
         var result = [];
+
         if (!webauthn.isAvailable()) {
           result.push(
             FormType.View(
-              { View: WindowsHelloErrorMessageView },
-              { selector: '.o-form-error-container' }
+              { View: new WindowsHelloErrorMessageView(
+                { message: Okta.loc('enroll.windowsHello.error.notWindows', 'login') }) },
+              { selector: '.o-form-error-container'}
             )
           );
         }
+        else {
+          result.push(FormType.View({
```

Comment:
@mauriciocastillosilva-okta , ah, I did misread - yes, I'm guessing there isn't any reason why we would diverge from how we're doing it elsewhere for this particular screen - @vincentvoong-okta, can you confirm that we should do it the normal way?